examples/react/webpack.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
 				options: {
 					loader: 'jsx',
 					target: 'es2015',
+					// This will make esbuild automatically generate import statements,
+					// making the ProviderPlugin unnecesary if used only for "react"
+					jsx: 'automatic',
```

Comment:
Add a comment that this setting is only available for React v17 & v16.14.0 with a link to https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html